docs/host_for_multiuser/setup.md

```diff
@@ -62,90 +64,97 @@ Follow the steps below to setup `multiuser` mode.
     - (keep databaseURL blank)
 7. Select "Service accounts" tab.
 8. Click "Generate new private key" in "Firebase Admin SDK" section.
-9. Put the downloaded file to `studio/config/auth/firebase_private.json`.
-
-### Setup mysql(or mariadb)
-1. Install mysql(or mariadb) server.
-2. Connect mysql server.
+9. Save the downloaded file to `studio/config/auth/firebase_private.json`.
+
+### Setup Database
+- Set up your own mysql (or mariadb) server or use docker compose mysql
+- Below are the instructions for using mysql with docker compose.
+
+1. Edit configs.
+    - Edit studio/config/.env
+      - Set `MYSQL_SERVER` to db server host or ip
+        - \*For docker platform, the fixed value `db:3306` is fine.
+      - Set `MYSQL_ROOT_PASSWORD` to {DB_ROOT_PASSWORD}, which you have decided.
+      - Set `MYSQL_DATABASE` to {YOUR_DATABASE_NAME}, which you have decided.
+      - Set `MYSQL_USER` to {DB_USER_NAME}, which you have decided.
+      - Set `MYSQL_PASSWORD` to {DB_USER_PASSWORD}, which you have decided.
+2. Install & run mysql server.
     ```bash
-    mysql -u root -p
-    ```
-3. Create database for your project.
-    ```sql
-    CREATE DATABASE YOUR_DATABASE_NAME;
+    docker compose -f docker-compose.dev.multiuser.yml up db -d
     ```
-4. Create user for your project.
-    ```sql
-    CREATE USER 'DB_USER_NAME'@'localhost' IDENTIFIED BY 'DB_USER_PASSWORD';
-    ```
-5. Grant all privileges to the user for the database.
-    ```sql
-    GRANT ALL PRIVILEGES ON YOUR_DATABASE_NAME.* TO 'DB_USER_NAME'@'localhost';
+    - The database and db_user are automatically generated based on the .env settings.
+3. Connect to mysql server.
+    ```bash
+    docker exec -it {DB_DOCKER_CONTAINER_NAME} mysql -u {DB_USER_NAME} -p {YOUR_DATABASE_NAME}
```

Comment:
@milesAraya 
> Where does the user find the DB_DOCKER_CONTAINER_NAME?

"DOCKER_CONTAINER" is listed, so I was aware that no explanation was necessary, but is it hard to understand?

> I think it may still be necessary to do these steps.

If you define the "MYSQL_*" environment variable in docker compose yaml, create database, create user, etc. will be executed automatically on first startup. (Is this not working in your environment?)

- Reference
  - https://hub.docker.com/_/mysql
    - > Initializing a fresh instance
  - https://stackoverflow.com/questions/43322033/create-database-on-docker-compose-startup

Reply:
Maybe I was doing it incorrectly, but I was getting the DOCKER_CONTAINER ID from Docker Desktop. 
Is there a simpler way? Where is "DOCKER_CONTAINER" listed?
<img width="183" alt="Screenshot 2024-06-26 at 14 07 41" src="https://github.com/arayabrain/barebone-studio/assets/171535965/934211bd-798c-4ba1-8200-265a62979c60">

Reply:
> Maybe I was doing it incorrectly,

Text has been adjusted. Please recheck.